WPMedia/fusion-news-theme-blocks
# THEMES-1538: Move useIdentity hook

## Description

This PR moves the useIdentity hook to Themes components to resolve dependency issues.

## Jira Ticket

- [THEMES-1538](https://arcpublishing.atlassian.net/browse/THEMES-1538)

## Acceptance Criteria

Hook is moved to Themes components

## Test Steps

1. Checkout this branch `git checkout THEMES-1538` for this repo and `arc-themes-components`
2. You may need to add `"@wpmedia/subscriptions-block@local"` to your `blocks.json` in the feature pack. Otherwise Fusion will try to also install the remote version of the block that is having the peer dependency issue and fail.
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/subscriptions-block,@wpmedia/identity-block`
4. Add each Identity block to test pages, and also the Subscription - Checkout block.
5. There should be no component errors for missing imports.

## Effect Of Changes

## Dependencies or Side Effects

- arc-themes-components: https://github.com/WPMedia/arc-themes-components/pull/221

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1538]: https://arcpublishing.atlassian.net/browse/THEMES-1538?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
